tokio/src/io/ready.rs

```diff
@@ -144,6 +163,23 @@ impl Ready {
         self.contains(Ready::WRITE_CLOSED)
     }
 
+    /// Returns `true` if the value includes priority `readiness`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use tokio::io::Ready;
+    ///
+    /// assert!(!Ready::EMPTY.is_priority_ready());
+    /// assert!(!Ready::WRITABLE.is_priority_ready());
+    /// assert!(Ready::PRIORITY.is_priority_ready());
+    /// ```
+    #[cfg(any(target_os = "linux", target_os = "android"))]
+    #[cfg_attr(docsrs, doc(cfg(any(target_os = "linux", target_os = "android"))))]
+    pub fn is_priority_ready(self) -> bool {
```

Comment:
This should probably just be `is_priority`. Being on the `Ready` struct already implies ready.